test/apps/CMakeLists.txt

```diff
@@ -0,0 +1,112 @@
+
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin/test)
+
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}
+    $<TARGET_PROPERTY:teca_alg,INTERFACE_INCLUDE_DIRECTORIES>
+    $<TARGET_PROPERTY:teca_io,INTERFACE_INCLUDE_DIRECTORIES>
+    )
+
+teca_add_test(test_deeplabv3p_ar_detect_app
+    LIBS teca_io teca_alg
```

Comment:
`LIBS` aren't needed since these tests aren't compiled (same comment applies where LIBS is passed and the test isn't compiled)

Reply:
Great point